src/reuse/_util.py

```diff
@@ -172,6 +184,68 @@ def _copyright_from_dep5(path: PathLike, copyright: Copyright) -> SpdxInfo:
     )
 
 
+def _parse_copyright_year(year: str) -> list:
+    """Parse copyright years and retun list."""
+    if not year:
+        ret = []
+    if re.match(r"\d{4}$", year):
+        ret = [int(year)]
+    if re.match(r"\d{4} - \d{4}$", year):
+        ret = [int(year[:4]), int(year[-4:])]
+    return ret
+
+
+def _merge_copyright_lines(copyright_lines: Set[str]) -> Set[str]:
```

Comment:
I don't remember the original intention here but I agree with you. Since the function is actually used by the main code path, it's better to make the naming explicit. I'll fix that.